doc/dev/setup.rst

```diff
@@ -0,0 +1,16 @@
+Development Setup
+=================
+
+Formatting and linting code
```

Comment:
This guide won't work if you don't also include the first part from FuseSoC that configures pre-commit: https://fusesoc.readthedocs.io/en/stable/dev/devsetup.html#setup-development-environment

Reply:
Done.